src/webhooks/webhook.ts

```diff
@@ -32,17 +32,24 @@ async function triggerAutomation(payload, record) {
   }
 
   const triggers: Record<string, (pr: any) => string> = {
-    closed: (pr) => (pr.merged ? "prMerged" : "prClosed"),
-    opened: (pr) => (pr.draft ? "draftPrOpened" : "prOpened"),
+    closed: (payload) => (payload.pull_request.merged ? "prMerged" : "prClosed"),
```

Comment:
this has changed so that the `pr` param is no longer used. I'd suggest moving this whole const triggers out of the function and changing the `pr` param to `payload` to match how the caller is using it

Reply:
Updated!